CMakeLists.txt

```diff
@@ -352,6 +356,10 @@ if (ENABLE_ALL OR ENABLE_PROMETHEUS OR NOT DISABLE_CIVET)
     list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/civetweb/dummy")
 endif()
 
+if (ENABLE_ALL OR ENABLE_GCP OR ENABLE_GRAFANA_LOKI)
+    include(Abseil)
+endif()
```

Comment:
Why do we need this here? Couldnt we include abseil twice? in gcp and grafana aswell (after the early returns ofcourse), since its a fetchcontent makeavailable we should be able to call it twice

Reply:
You are right, updated in 407abbc65a8900dc9937dc0587c52d68693335ff